docs/sources/ingest-and-analyze-profile-data/pyroscope-ui.md

```diff
@@ -1,35 +1,120 @@
 ---
-title: Use Pyroscope UI
+title: Use Pyroscope UI to explore profiling data
 menuTitle: Use the Pyroscope UI
-description: How to use the Pyroscope UI to analyze performance of your applications.
+description: How to use the Pyroscope UI to explore profile data.
 weight: 40
 keywords:
   - pyroscope
   - performance analysis
   - flamegraphs
 ---
 
-## Pyroscope: Continuous Profiling in Action
-
-![Screenshots of Pyroscope's UI](https://grafana.com/static/img/pyroscope/pyroscope-ui-diff-2023-11-30.png)
+# Use the Pyroscope UI to explore profiling data
 
 Pyroscope's UI is designed to make it easy to visualize and analyze profiling data.
 There are several different modes for viewing, analyzing, uploading, and comparing profiling data.
-We will go into more detail about these modes in the [Pyroscope UI documentation].
-For now, it's important to note that one of the major benefits of continuous profiling is the ability to compare and diff profiling data from two different queries:
 
-- Comparing two different git commits before and after a code change
-- Comparing Staging vs production environments to identify differences in performance
-- Comparing performance between two different a/b tests or feature flag experiments
-- Comparing memory allocations between two different time periods before and after a memory leak
-- etc
+![Screenshots of Pyroscope's UI](https://grafana.com/static/img/pyroscope/pyroscope-ui-diff-2023-11-30.png)
+
+While code profiling has been a long-standing practice, continuous profiling represents a modern and more advanced approach to performance monitoring. This technique adds two critical dimensions to traditional profiles:
+
+Time
+: Profiling data is collected _continuously_, providing a time-centric view that allows querying performance data from any point in the past.
```

Comment:
This markdown format creates a term and definition list like this: 

Term 
: Definition